packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -690,6 +690,81 @@ void main() {
     await tester.tap(find.text('Menu Button'));
   });
 
+  testWidgets('PopupMenuItem child height is a minium, child is vertically centered', (WidgetTester tester) async {
```

Comment:
Some wording suggestions: 
```suggestion
  testWidgets('PopupMenuItem child height represents minimum widget height, child is vertically centered', (WidgetTester tester) async {
```

It might also be better if the minimum height and vertically centered tests are separated since these seem like unique behaviors

Reply:
Most tests implicitly or explicitly cover a host of related dimensions. I think that's OK.